docs/devGuide/githubActions/markbindAction.md

```diff
@@ -0,0 +1,33 @@
+{% set title = "GitHub Actions: markbind-action" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: default
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+A GitHub Action that builds and deploys your MarkBind site. The source code is at [MarkBind/markbind-action](https://github.com/MarkBind/markbind-action).
```

Comment:
Maybe can add a brief explanation on the purpose of this repo? (help users to streamline their workflow, easier for them to deploy without having to write their own actions from scratch, make their workflows consistent etc)